### PR TITLE
CASMINST-5988

### PR DESCRIPTION
**Pull request by atifsyedali**
# Description

[CASMINST-5988](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5988) BREAK/FIX: management-node-rollout breaks with "default" product containing jinja templates

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: workflows/iuf/stages.yaml [hunk 1]
```diff
@@ -83,14 +83,17 @@ stages:
         static-parameters: {} # any parameters that will be supplied statically to this operation.
         required-manifest-attributes:
           - content.vcs
+        include-default-product-in-site-params: true # this operation wants to receive site_params.products.default in supplied global_params
```
Comment (haasken-hpe) [inline workflows/iuf/stages.yaml:86]: So for all the other operations, this defaults to `false` somehow, right?

**Review by haasken-hpe — APPROVED** [review]

File: workflows/iuf/stages.yaml (hunk 1 above)
Comment (haasken-hpe) [reply workflows/iuf/stages.yaml:86]: I confirmed with Atif earlier today that this does default to false.

> Ryan: how does defaulting the include-default-product-in-site-params work? Is it just using the default value for an empty value of type bool in golang?
> Atif: yes. default is false

**Review by rustydb — APPROVED** [review]

**Pull request merged**